ohhell.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import List
+
+
+
+DECK_SIZE = 54
```

Comment:
Nit: `SCORE_MULTIPLIER` should be before the `GAME_VERSION` file opening/writing stuff

Reply:
also why is deck size 54?

Reply:
Yeah all standard deck sizes are 52. Please fix!!!!